listener.go

```diff
@@ -132,7 +146,7 @@ func (h *hub) processReqDequeue(req baps3.Message) (resps []*baps3.Message) {
 }
 
 func (h *hub) processReqEnqueue(req baps3.Message) (resps []*baps3.Message) {
-	args := req.AsSlice()[1:]
+	args := req.Args()
 	if len(args) != 4 {
 		return append(resps, baps3.NewMessage(baps3.RsWhat).AddArg("Bad command"))
```

Comment:
Easiest way to check:  how much shorter would it be to make one function that's used five times, over repeating the code five times?  how much easier would changes to the bad-command logic?  5 isn't exactly the smallest of numbers when we're talking about code repetition.

At the very least, `"bad command"` should be a constant.  We're probably going to change the error message format at some point to add machine readability anyway...